client/my-sites/people/people-invite-details/test/index.jsx

```diff
@@ -150,11 +150,6 @@ describe( 'PeopleInviteDetails', () => {
 		expect( mockGoBack ).toHaveBeenCalledTimes( 1 );
 		expect( mockGoBack ).toHaveBeenCalledWith( '/people/invites/' + siteObject.slug );
 
-		// Verify that a placeholder is rendered while waiting for `page.back`
```

Comment:
Since we are refreshing the list again after revoking or inviting, we don't need this placeholder anymore.
The Placeholder was causing issues when revoking and inviting again the same user.